mne_bids/tests/test_utils.py

```diff
@@ -219,10 +219,10 @@ def test_parse_ext():
 
 
 @pytest.mark.parametrize('fname', [
-    'sub-01_ses-02_task-test_run-3_meg.fif',
-    'sub-01_ses-02_task-test_run-3.fif',
-    'sub-01_ses-02_task-test_run-3',
-    '/bids_root/sub-01/ses-02/meg/sub-01_ses-02_task-test_run-3_meg.fif',
+    'sub-01_ses-02_task-test_run-3_part-01_meg.fif',
+    'sub-01_ses-02_task-test_run-3_part-01.fif',
+    'sub-01_ses-02_task-test_run-3_part-01',
```

Comment:
@jasmainak, do you know why this and the line above it is being tested. neither are valid bids-compatible filenames

Reply:
to allow for making bids_basename?